psi4/src/psi4/libfock/cubature.cc

```diff
@@ -5021,5323 +4726,7 @@ void SphericalGrid::initialize_lebedev()
-    v=0.3586067974412447E-3;
-    start = lebedev_reccurence(3,start,a,b,v,s);
-    a=0.1886108518723392E-1;
-    v=0.9831528474385880E-4;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.4800217244625303E-1;
-    v=0.1605023107954450E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.8244922058397242E-1;
-    v=0.2072200131464099E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.1200408362484023E+0;
-    v=0.2431297618814187E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.1595773530809965E+0;
-    v=0.2711819064496707E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.2002635973434064E+0;
-    v=0.2932762038321116E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.2415127590139982E+0;
-    v=0.3107032514197368E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.2828584158458477E+0;
-    v=0.3243808058921213E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.3239091015338138E+0;
-    v=0.3349899091374030E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.3643225097962194E+0;
-    v=0.3430580688505218E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.4037897083691802E+0;
-    v=0.3490124109290343E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.4420247515194127E+0;
-    v=0.3532148948561955E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.4787572538464938E+0;
-    v=0.3559862669062833E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.5137265251275234E+0;
-    v=0.3576224317551411E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.5466764056654611E+0;
-    v=0.3584050533086076E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.6054859420813535E+0;
-    v=0.3584903581373224E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.6308106701764562E+0;
-    v=0.3582991879040586E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.6530369230179584E+0;
-    v=0.3582371187963125E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.6718609524611158E+0;
-    v=0.3584353631122350E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.6869676499894013E+0;
-    v=0.3589120166517785E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.6980467077240748E+0;
-    v=0.3595445704531601E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.7048241721250522E+0;
-    v=0.3600943557111074E-3;
-    start = lebedev_reccurence(4,start,a,b,v,s);
-    a=0.5591105222058232E-1;
-    v=0.1456447096742039E-3;
-    start = lebedev_reccurence(5,start,a,b,v,s);
-    a=0.1407384078513916E+0;
-    v=0.2252370188283782E-3;
-    start = lebedev_reccurence(5,start,a,b,v,s);
-    a=0.2364035438976309E+0;
-    v=0.2766135443474897E-3;
-    start = lebedev_reccurence(5,start,a,b,v,s);
-    a=0.3360602737818170E+0;
-    v=0.3110729491500851E-3;
-    start = lebedev_reccurence(5,start,a,b,v,s);
-    a=0.4356292630054665E+0;

```

Comment:
There are two lebedev builders, this one was not used. I think the other looks a bit cleaner as well.